src/ReactiveDomain.Users/Services/ActiveDirectoryUserSearch.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.DirectoryServices.AccountManagement;
+using System.Linq;
+
+namespace ReactiveDomain.Users.Services
+{
+    public static class ActiveDirectoryUserSearch
+    {
+        public static List<Principal> FindUserPrincipal(string userName, PrincipalContext principalContext)
+        {
+            List<Principal> results = new List<Principal>();
+            using (var searcher = new PrincipalSearcher(new UserPrincipal(principalContext)))
+            {
+                //PrincipalSearcher for machine for local users is case sensitive, this is a bug with .net. So we have below workaround for local user search.
+                //For more details on this refer to https://github.com/dotnet/corefx/issues/26779
+                if (principalContext.ContextType == ContextType.Machine)
+                {
+                    // Remove the leading & trailing * (this comes when called from Elbe.WPF)
+                    userName = userName.Trim('*');
+                    var users = searcher.FindAll().Where(x =>
+                    {
+                        // .net 4.8 doesn't have case insensitive version of Contains(), but dotnet core does. Since Elbe is targeted for both frameworks, I had the use IndexOf to achieve
```

Comment:
Let's remove references to "Elbe"
```suggestion
                        // Framework 4.8 doesn't have case insensitive version of Contains(), but .NET does. Since we are targeting both .NET and Framework, I had the use IndexOf to achieve
```

Reply:
:+1: